package/cpp/api/JsiSkCanvas.h

```diff
@@ -534,6 +534,17 @@ class JsiSkCanvas : public JsiSkHostObject {
   void setCanvas(SkCanvas *canvas) { _canvas = canvas; }
   SkCanvas *getCanvas() { return _canvas; }
 
+  /**
+   Returns the underlying object from a host object of this type
+  */
+  static SkCanvas *fromValue(jsi::Runtime &runtime,
```

Comment:
As the API design is to call `animation.render(canvas, progress)` I needed to get the canvas as parameter when called from JS